galleries/examples/subplots_axes_and_figures/subplots_demo.py

```diff
@@ -133,7 +133,7 @@
 # same scale when using ``sharey=True``.
 
 fig, axs = plt.subplots(3, sharex=True, sharey=True)
-fig.suptitle('Sharing both axes')
+fig.suptitle('Sharing both Axes')
```

Comment:
I think this is referring to the plural of `x and y axes`, not the `Axes` object, and thus should stay lower case?

Reply:
`x- and y-axis` could be used, as it is more specific, and the hyphenation avoids the need to pluralize.